source/fundamentals/stable-api.txt

```diff
@@ -0,0 +1,130 @@
+.. _csharp-stable-api:
+
+==============
+{+stable-api+}
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. note::
+
+   The {+stable-api+} feature requires MongoDB Server 5.0 or later.
+
+   You should use the {+stable-api+} feature only if all the MongoDB
+   servers you're connecting to support this feature.
+
+Overview
+--------
+
```

Comment:
Could you add an "in this guide" statement? Here is the one from Godoc:

"In this guide, you can learn how to specify **Stable API** compatibility when connecting to a MongoDB instance or replica set."